### ISSUE TITLE
Invalid Input when doing a getPageById

### ISSUE THREAD

**Issue by RaphArbuz** [Frexeptabel/notion-api#16]
Hi,

Thank you for building this module.

getPages() returns a list of pageIds, but getPageById, for any of them, returns:

```
{
  errorId: '8e55a984-004f-44a3-8338-a7c68617e9e4',
  name: 'ValidationError',
  message: 'Invalid input.'
}
```

Any idea why? I need to be able to fetch Notion tables with an API.

Many thanks!

**Comment by nurhsans** [comment]
hi, I get this error too after using my notion page's pageId as well. Manage to return the page html and attributes when I add the two other parameters to the getPageById method.

**Comment by LandonSchropp** [comment]
@picanteverde Is there any update on fixing this one? I'm running into it as well, and it looks like @Allgoll put out a PR for this.

**Comment by LandonSchropp** [comment]
I tried the API change in #18 locally, but unfortunately, I still seem to be experiencing this problem. Adding a couple of addition attributes to `getPageById` didn't seem to help.

**Comment by Allgoll** [comment]
@LandonSchropp it's strange. I'm using my own fork with #18 inside and my application works without this error. Maybe it's another error? Or maybe you missed something?

**Comment by LandonSchropp** [comment]
@Allgoll I took another look and I think I did two things wrong here:

* The first time around, I applied your changes to my locally downloaded dist. This time, I downloaded your repo, built, and linked it directly. I think I must have accidentally edited the source instead of the dist when doing it myself locally.
* The UUID of the data *must* contain hyphens. If It doesn't, the call with fail with a cryptic error.

My bad. 🙂 Thanks for the help!
